doc/onboarding-extras.md

```diff
@@ -13,6 +13,7 @@
 | `lib/{crypto,tls,https}` | @nodejs/crypto |
 | `lib/domains` | @misterdjules |
 | `lib/fs`, `src/{fs|file}` | @nodejs/fs |
+| `lib/internal/url` | @nodejs/url |
```

Comment:
That's going to be confusing/unclear if there's not a separate entry for `lib/url` in addition to the entry for `lib/internal/url`. Not opposed to this landing as is, but if there's different people to cc depending on which one you're working on, I'd *prefer* that there be entries for both.